extension.js

```diff
@@ -493,17 +488,20 @@ class GEWallpaperIndicator extends panelMenu.Button {
     }
 });
 
-function init(extensionMeta) {
-    ExtensionUtils.initTranslations("GoogleEarthWallpaper");
-}
-
-function enable() {
-    googleearthWallpaperIndicator = new GEWallpaperIndicator();
-    main.panel.addToStatusArea(IndicatorName, googleearthWallpaperIndicator);
-}
+export default class GoogleEarthWallpaper extends Extension {
+    enable() {
+        if (!globalThis.GoogleEarthWallpaperState) globalThis.GoogleEarthWallpaperState = {
+            gettext : _,
+            PACKAGE_VERSION
+        };
```

Comment:
importing these resources from outside of `extension.js`/`prefs.js` doesn't seem to work, so I pass them around globally to modules higher in the dependency tree instead.